src/test/java/org/gridsuite/filter/server/expertrule/NumberExpertRuleTest.java

```diff
@@ -442,35 +446,58 @@ private static Stream<Arguments> provideArgumentsForLoadTest() {
                 // VoltageLevel fields
                 Arguments.of(EQUALS, FieldType.NOMINAL_VOLTAGE, 13.0, null, load, true),
                 Arguments.of(EQUALS, FieldType.NOMINAL_VOLTAGE, 14.0, null, load, false),
+                // Load fields
+                Arguments.of(EQUALS, FieldType.P0, 77.0, null, load, true),
+                Arguments.of(EQUALS, FieldType.Q0, 277.0, null, load, true),
+                Arguments.of(EQUALS, FieldType.P0, 50.0, null, load, false),
+                Arguments.of(EQUALS, FieldType.Q0, 300.0, null, load, false),
 
                 // --- GREATER_OR_EQUALS --- //
                 // VoltageLevel fields
                 Arguments.of(GREATER_OR_EQUALS, FieldType.NOMINAL_VOLTAGE, 12.0, null, load, true),
                 Arguments.of(GREATER_OR_EQUALS, FieldType.NOMINAL_VOLTAGE, 13.0, null, load, true),
                 Arguments.of(GREATER_OR_EQUALS, FieldType.NOMINAL_VOLTAGE, 14.0, null, load, false),
+                // Load fields
+                Arguments.of(GREATER_OR_EQUALS, FieldType.P0, 72.0, null, load, true),
```

Comment:
Each field P0, Q0 possible to add 3 test cases as the order of NOMINAL_VOLTAGE for the homogeneity
For GREATER_OR_EQUALS operator, it is better to test also equal case

Reply:
Done.